package.json

```diff
@@ -15,42 +15,44 @@
   ],
   "private": true,
   "dependencies": {
-    "@angular-devkit/core": "^0.6.8",
-    "@angular/animations": "7.2.4",
-    "@angular/common": "7.2.4",
-    "@angular/compiler": "7.2.4",
-    "@angular/core": "7.2.4",
-    "@angular/forms": "7.2.4",
-    "@angular/platform-browser": "7.2.4",
-    "@angular/platform-browser-dynamic": "7.2.4",
-    "@angular/router": "7.2.4",
-    "core-js": "^2.4.1",
-    "marked": "^0.3.9",
-    "rxjs": "^6.4.0",
-    "tslib": "^1.9.0",
-    "zone.js": "^0.8.29"
+    "@angular-devkit/core": "next",
```

Comment:
right, I set them to next because it helps not having to update them from one beta version to the other, right now the latest is 8.0.0-beta.13, we could wait until v8 is finally released

Reply:
Angular 8.0.0 is finally released so I pinned the versions again